elasticsearch/templates/secret-cert.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.createCert }}
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: {{ template "elasticsearch.name" . }}-certs
+  labels:
+    app: {{ template "elasticsearch.name" . }}
+    chart: "{{ .Chart.Name }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
```

Comment:
Is this hook really needed?

Reply:
yes, it will guarantee the certs will be created only on chart install